core/src/main/java/org/verapdf/policy/SchematronPipeline.java

```diff
@@ -45,6 +45,15 @@ final class SchematronPipeline {
 	private static final Templates cachedExpXsl = createCachedTransform(isoExpXsl);
 	private static final Templates cachedIsoSvrlXsl = createCachedTransform(isoSvrlXsl);
 
+	static {
+		try {
+			factory.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
+			factory.setAttribute(XMLConstants.ACCESS_EXTERNAL_STYLESHEET, "file");
+		} catch (TransformerConfigurationException ignored) {
+			LOGGER.log(Level.WARNING, "Unable to secure xsl transformer");
```

Comment:
Is a warning log message enough here? It might be better to generate a signal that could let the CLI or GUI know so they can inform the user. Most GUI users will never see the log message I guess.

Reply:
As I understand it, this exception throws in the case of unsupported features and attributes, and for FEATURE_SECURE_PROCESSING and ACCESS_EXTERNAL_STYLESHEET it should never happen